### DISCUSSION TITLE
Extracting webm to opus

### DISCUSSION THREAD

**Discussion by moonman** [kannagi0303/yt-dlp-gui#53] [Q&A]
Hi I am wondering if it's possible to extract the downloaded .webm file into .opus. I download mostly audio and yt-dlp has a parameter "-x" to extract the auido from the container and it works well via cmd, but I see no such option in the gui. Is there a way to specify this in the config yaml?

**Comment by kannagi0303** [comment]
I don't know if that will meet your needs.

On the right side of the format's Audio, there is a small button that can be downloaded separately.

**Reply by moonman** [reply]
Yep, that's the one I'm using. It does download just the audio, but the audio is wrapped into webm container. When "-x" is passed to yt-dlp on the command line it actually extracts the audio into just .opus (or aac - whichever audio you download). There is no options to "extract" in the gui though. The audio just stays in the webm container.

**Reply by kannagi0303** [reply]
I understand your needs.
There is indeed no corresponding feature at present. I will add the option in a future version

**Reply by kannagi0303** [reply]
Hello, sorry to keep you waiting. I have released a new version which allows you to save Audio as .opus, .acc, .mp3, and more. If you're interested, please feel free to give the new version a try.
